exercise/migrations/0009_merge_20190930_1445.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.5 on 2019-09-30 14:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('exercise', '0008_auto_20190927_1338'),
+        ('exercise', '0006_auto_20190930_1018'),
+    ]
+
+    operations = [
+    ]
```

Comment:
could you squash these new migrations into one pls?

Reply:
Done.